.github/workflows/performance.yml

```diff
@@ -123,5 +123,19 @@ jobs:
       - name: Print Performance tests results
         run: "node ./tests/performance/results.js"
 
+      - uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975 # v6.4.0
+        id: commit-timestamp
+        with:
+            github-token: ${{secrets.GITHUB_TOKEN}}
+            script: |
+                const commit_details = await github.rest.git.getCommit({owner: context.repo.owner, repo: context.repo.repo, commit_sha: context.sha});
+                return parseInt((new Date( commit_details.data.author.date ).getTime() / 1000).toFixed(0))
+      - name: Publish performance results
+        env:
+            COMMITTED_AT: ${{ steps.commit-timestamp.outputs.result }}
+            CODEHEALTH_PROJECT_TOKEN: ${{ secrets.CODEHEALTH_PROJECT_TOKEN }}
+            HOST_NAME: "codehealth-test.vercel.app"
```

Comment:
We'll need to update this to point to the production dashboard once it's ready for us to start pushing data into it.

Reply:
We could consider adding a `TODO` comment to it maybe?

Reply:
Thanks for review. `TODO` comment added.